services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -1145,13 +1146,36 @@ Optional<String> getTopicForCurrentPushJob(
       latestTopic = Optional.of(versionTopics.get(0));
     }
 
+    Map<String, String> futureVersions = getFutureVersionsForMultiColos(clusterName, storeName);
+    for (Map.Entry<String, String> entry: futureVersions.entrySet()) {
+      ControllerClient controllerClient = getVeniceHelixAdmin().getControllerClientMap(clusterName).get(entry.getKey());
+      StoreResponse storeResponse = controllerClient.retryableRequest(5, c -> c.getStore(storeName));
```

Comment:
Is there any rationale behind using `5` as the retry count here as opposed to `10` below for the `getFutureVersions`?

Reply:
I assumed, getStore is called more often hence higher chance of it having cached.